### DISCUSSION TITLE
Update image preview for older models

### DISCUSSION THREAD

**Discussion by Jeefrect** [BlafKing/sd-civitai-browser-plus#54] [Ideas]
Hello, I am using your extension and it works very well indeed. **Thank you!**

It would be great to see automatic image uploading for already installed models. For example I put your extension when I already had over 200 lora and checkpoint models... All of them without images(previews), and it would be great to update the images for these models in a couple of clicks!

**Comment by BlafKing** [comment]
That should definitely be possible, and I'm glad to hear that you like the extension!

I'll make sure to start working on that after the upcoming update that redesigns the UI :)

**Comment by mosherc** [comment]
[The official Civit AI extension](https://github.com/civitai/sd_civitai_extension) does this already (automatically).

**Comment by BlafKing** [comment]
Took me way too long since I forgot about this request (sorry about that)
It has now been implemented in v1.16! :)

**Reply by Jeefrect** [reply]
thanks!
